packages/enzyme-adapter-react-17/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "name": "enzyme-adapter-react-17",
+  "version": "0.0.0",
+  "description": "JavaScript Testing utilities for React",
+  "homepage": "https://enzymejs.github.io/enzyme/",
+  "main": "build",
+  "scripts": {
+    "clean": "rimraf build",
+    "lint": "eslint --ext js,jsx .",
+    "pretest": "npm run lint",
+    "prebuild": "npm run clean",
+    "build": "babel --source-maps=both src --out-dir build",
+    "watch": "npm run build -- -w",
+    "prepublish": "not-in-publish || (npm run build && safe-publish-latest && cp ../../{LICENSE,README}.md ./)"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/enzymejs/enzyme.git",
+    "directory": "packages/enzyme-adapter-react-17"
+  },
+  "keywords": [
+    "javascript",
+    "shallow rendering",
+    "shallowRender",
+    "test",
+    "reactjs",
+    "react",
+    "flux",
+    "testing",
+    "test utils",
+    "assertion helpers",
+    "tdd",
+    "mocha"
+  ],
+  "author": "Jordan Harband <ljharb@gmail.com>",
+  "funding": {
+    "url": "https://github.com/sponsors/ljharb"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "enzyme-adapter-utils": "^1.13.1",
```

Comment:
Even `^1.14.0` (which is currently `latest`) will throw peerDependency warnings due to requiring `react` `0.13.x || 0.14.x || ^15.0.0-0 || ^16.0.0-0`. 

`enzyme-adapter-utils` needs to have peerDependencies updated.
`airbnb-prop-types` needs to have peerDependencies updated.

Reply:
Hmm, that is a bit tricky, since `airbnb-prop-types` also uses enzyme to test itself, and it can't really claim to support React 17 unless it tests on it.

I'll be sure to update the peer deps for all the enzyme packages in this PR, though.

Reply:
I don't think that needs to be much of a concern. `react` is only a peerDependency in `airbnb-prop-types`.

Reply:
Yes, which means that `npm ls` will fail when airbnb-prop-types and react 17 are installed, and npm 7+ will fail also.